assets/i18n/qqq.json

```diff
@@ -506,8 +506,9 @@
 	"I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_2": "Text on the Teach page outlining Oppia's Publication Policy.",
 	"I18N_TEACH_PAGE_TAB_PARTICIPATION_PLAYBOOK": "Text on the tab for Participation Playbook on the Teach page.",
 	"I18N_TEACH_PAGE_TAB_TEACH_WITH_OPPIA": "Text on the tab for Teach with Oppia on the Teach page.",
-	"I18N_TOPIC_VIEWER_PRACTICE": "Title shown in a tab header in the topic's viewer page. - Clicking on this tab header shows a list of the subtopics that lie within the topic.\n{{Identical|Practice}}",
-	"I18N_TOPIC_VIEWER_STORY": "Title shown in a tab header in the topic's viewer page. - Clicking on this tab header shows a list of the stories that lie within the topic.\n{{Identical|Story}}",
+	"I18N_TOPIC_VIEWER_PLAY": "Title shown in a tab header in the topic's viewer page. - Clicking on this tab header shows a list of the stories that lie within the topic.",
+	"I18N_TOPIC_VIEWER_REVIEW": "Title shown in a tab header in the topic's viewer page. - Clicking on this tab header shows a list of the subtopics that lie within the topic.",
+	"I18N_TOPIC_VIEWER_TRAIN": "Title shown in a tab header in the topic's viewer page. - Clicking on this tab header shows a list of the practice questions that can be used to review the topic.",
```

Comment:
Please change "review" to "test the learner's understanding of", to avoid confusion with the Review tab.

Reply:
Done